relay-base-schema/src/data_category.rs

```diff
@@ -124,6 +124,14 @@ pub enum DataCategory {
     SeerAutofix = 27,
     /// This is the data category to count Seer Scanner run events.
     SeerScanner = 28,
+    /// CodecovSeat
+    ///
+    /// This is the data category to count assigned Codecov seats.
+    CodecovSeat = 29,
```

Comment:
In this file there is a `TryFrom` implementation and a test that you will have to update as well. The test is supposed to fail, but apparently it's not written very well.